### ISSUE TITLE
+ Code + Markdown weird design

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#93953]
Refs: #93742

I find the + Code + Markdown separator design a bit weird.
We do not use it anywhere else in vscode and it is not accessible - no way to focus into it.
I would remove it until other parts of the notebook ux are polished.
And this could ideally be done via context menus

![Screenshot 2020-03-31 at 16 19 58](https://user-images.githubusercontent.com/1926584/78037314-a844d680-736b-11ea-9c32-8a09f018e117.png)

**Comment by rebornix** [comment]
Let's look into this issue next iteration.

**Comment by rebornix** [comment]
We now allow tab into the toolbar with c22cc2b3b6cf95f9d8f4ff816f6ef947c1303ec5

**Comment by jrieken** [comment]
This has been changed to be framed and thereby looks more like buttons
